.travis.yml

```diff
@@ -1,5 +1,11 @@
 sudo: false
+dist: precise
 language: java
+addons:
+  # https://github.com/travis-ci/travis-ci/issues/5227#issuecomment-165131913
+  hosts:
+    - fakehostnametoworkaroundtravisbug
```

Comment:
please put some dashes in the `fake-host-name-to-work-around-travis-bug`, if that does not break it